src/sources/forus-platform/js/angular-1/i18n/nl/pages/voucher-records.pug.i18n.js

```diff
@@ -1,17 +1,17 @@
 module.exports = {
     header: {
-        title: "Records",
+        title: "Persoonlijke eigenschappen",
     },
     labels: {
         id: 'ID',
-        record_type: 'Record type',
-        value: 'Value',
-        created_at: 'Date',
-        note: 'Note',
-        action: 'Actions',
-        search: 'Search',
+        record_type: 'Soort eigenschap',
+        value: 'Eigenschap',
+        created_at: 'Datum',
```

Comment:
@GerbenBosschieter was confused by this original naming, because was not clear is it date of creation, date of update or something else. lets set here creation date?